pci/src/bus.rs

```diff
@@ -1,6 +1,7 @@
 // Copyright 2018 The Chromium OS Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE-BSD-3-Clause file.
+// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
```

Comment:
Wrong license. I guess it should be `BSD-3-Clause` as no copyright header is in place other than the initial one from Crosvm? @rbradford

Reply:
If we've modified it (which I guess we have - check the git log) then it should be "AND Apache 2.0"